src/components/simple-color-picker/simple-color-picker.spec.tsx

```diff
@@ -403,16 +489,19 @@ describe("SimpleColorPicker", () => {
   });
 
   describe("events", () => {
-    let wrapper, onBlur, documentMousedownCallback, fireDocumentMousedown;
+    let wrapper: ReactWrapper;
+    let onBlur: jest.Mock;
+    let documentMousedownCallback: EventListener;
+    let fireDocumentMousedown: () => void;
 
     beforeEach(() => {
       document.addEventListener = jest.fn((eventName, callback) => {
         if (eventName === "mousedown") {
-          documentMousedownCallback = callback;
+          documentMousedownCallback = callback as EventListener;
         }
       });
       fireDocumentMousedown = () => {
-        const customEvent = { target: document };
+        const customEvent = ({ target: document } as unknown) as Event;
```

Comment:
this double-assertion through `unknown` is pretty ugly. I don't have an easy suggestion but it feels like there should be another better way of doing this? No worries though if you already spent time on this and couldn't find one (as it's only a test file and one we'll eventually be replacing with RTL).

Reply:
AFAIK theres unfortunately no other way to mock the `target` of the `Event` so I had to brute force mock it  :)
https://developer.mozilla.org/en-US/docs/Web/API/Event/Event